src/debugpy/common/log.py

```diff
@@ -359,7 +359,7 @@ def report_paths(get_paths, label=None):
         report("Installed packages:\n")
         try:
             for pkg in importlib_metadata.distributions():
-                report("    {0}=={1}\n", pkg.name, pkg.version)
+                report("    {0}=={1}\n", pkg.metadata['Name'], pkg.version)
```

Comment:
Does this actually fix the issue when name is missing? Looking at implementation of `importlib.metadata.Distribution`, `pkg.name` just calls `pkg.metadata['Name']`, so the effect should be the same.